### DISCUSSION TITLE
windows下密码乱码

### DISCUSSION THREAD

**Discussion by sw586** [AlistGo/alist#1218] [Q&A]
![image](https://user-images.githubusercontent.com/24788222/173185631-b94b6909-2769-4b50-969f-6d3c7eaf29d5.png)
![image](https://user-images.githubusercontent.com/24788222/173185676-c60adea5-ee19-4e50-846e-c4166c3315c7.png)

这种 密码是多少  第一次安装使用

**Comment by anwen-anyi** [comment]
你的密码是 `KSL9E6Cx`
